ani-cli

```diff
@@ -318,7 +322,11 @@ use_external_menu="${ANI_CLI_EXTERNAL_MENU:-0}"
 [ "$use_external_menu" = "0" ] && multi_selection_flag="${ANI_CLI_MULTI_SELECTION:-"-m"}"
 [ "$use_external_menu" = "1" ] && multi_selection_flag="${ANI_CLI_MULTI_SELECTION:-"-multi-select"}"
 cache_dir="${ANI_CLI_CACHE_DIR:-${XDG_CACHE_HOME:-$HOME/.cache}/ani-cli}"
-[ ! -d "$cache_dir" ] && mkdir -p "$cache_dir"
+if [ ! -d "$cache_dir" ]; then
+    mkdir -p "$cache_dir"
+else
+    rm -f "$cache_dir/*"
+fi
```

Comment:
No?
Cache dir should be fine to be persistent. 
Also with this you're either initializing the cache dir or removing it, which results in inconsistent behavior.
Also there doesn't need to be a `[ ! -d "$cache_dir" ]` because `mkdir -p` is already idempotent.

Reply:
it's not removing cache_dir it's removing the files in it if exist

Reply:
![image](https://github.com/pystardust/ani-cli/assets/76195824/efa22f61-4ca4-4bbd-bf67-6b2b00f0ef44)

Reply:
still odd. If the files are truly so irrelevant to be removed after every run, why not use `mktemp`?

Reply:
it's because sometimes links are not generated, and it starts playing links from dangling file